api/zeebe/generation-strategy.js

```diff
@@ -22,12 +23,13 @@ function hackChangesetDescription(specPath) {
   // This adjustment replaces the description of the `Changeset` schema with the current description of
   //   the `UserTaskUpdateRequest.changeset` property.
   console.log("hacking changeset description...");
-  replace.sync({
-    files: `${specPath}`,
-    from: /^      description: A map of changes.$/m,
-    to: `      description: |
-        JSON object with changed task attribute values.
 
+  const content = fs.readFileSync(specPath, "utf8");
+
+  const updated = content.replace(
+    /^      description: A map of changes\.$/m,
```

Comment:
```suggestion
    /^      description: A map of changes.$/m,
```
Not sure if this behaves differently.
But the `.` was not escaped in the prior regex.

Reply:
ah yes, my bad 🤦  Thank for catching!